bindsnet/network/topology.py

```diff
@@ -72,10 +72,14 @@ def update(self, **kwargs):
         Compute connection's update rule.
         '''
         reward = kwargs.get('reward', None)
+        mask = kwargs.get('mask', None)
 
         if self.update_rule is not None:
             self.update_rule(self, reward=reward)
 
+        if mask is not None:
+            self.w.masked_fill_(mask, 0)
```

Comment:
Please describe in the docs what mask is, what is input size, like you did for reward